Content.Shared/_NF/Trade/TradeCrateComponent.cs

```diff
@@ -1,9 +1,58 @@
-namespace Content.Shared._NF.Trade.Components;
+using System.Threading;
+using Content.Shared.Cargo;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared._NF.Trade;
 
 /// <summary>
-/// This is used to mark an entity to be used in a trade crates
+/// This is used to mark an entity to be used as a trade crate
 /// </summary>
-[RegisterComponent]
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState, AutoGenerateComponentPause, Access(typeof(SharedCargoSystem))]
 public sealed partial class TradeCrateComponent : Component
 {
+    /// <summary>
+    /// The value of the crate, in spesos, when at its destination.
+    /// </summary>
+    [DataField(serverOnly: true)]
+    public int ValueAtDestination;
+
+    /// <summary>
+    /// The value of the crate, in spesos, when delivered elsewhere.
+    /// </summary>
+    [DataField(serverOnly: true)]
+    public int ValueElsewhere;
+
+    /// <summary>
+    /// If non-zero, this crate will be an express delivery.
+    /// </summary>
+    [DataField(serverOnly: true, customTypeSerializer: typeof(TimeOffsetSerializer))]
```

Comment:
doesnt need a serializer because this is only set once in yaml and never changes